### DISCUSSION TITLE
SignalR

### DISCUSSION THREAD

**Discussion by dustan32** [PrismLibrary/Prism#2921] [General]
For the great people here either using or working with Prism, i have a query and can't seem to find any answers online.

Is it possible to use SignalR within a WPF prism application. 

Any pointers/help would be hugely appreciated. 

Thanks in advance!

**Comment by dansiegel** [comment]
Yes it should absolutely be possible to do.. but Prism itself has very little to do with your setup there other than the fact that you'll likely want to register some things with the Container.
